get-modified-source-files/action.yml

```diff
@@ -0,0 +1,30 @@
+name: "get-modified-source-files"
+description: "Get modified source files"
+
+outputs:
+  modified-source-files:
+    description: "Modified source files"
+    value: ${{ steps.get-modified-source-files.outputs.modified-source-files }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        # Fetch with depth=0 to find modified files
+        fetch-depth: 0
+
+    - name: Get modified source files
+      id: get-modified-source-files
+      run: |
+        # List added/modified/renamed files excluding deleted ones
+        modified_files=$(git --no-pager diff --name-only --diff-filter=d origin/"${{ github.base_ref }}"...HEAD)
+        modified_source_files=$(echo "$modified_files" | egrep "\.(cpp|hpp)$") || true
+        echo ::set-output name=modified-source-files::${modified_source_files}
+      shell: "bash"
+
+    - name: Show result
+      run: |
+        echo "modified source files: ${{ steps.get-modified-source-files.outputs.modified-source-files }}"
+      shell: "bash"
```

Comment:
Let's make it consistent whether we should use `""` or not.
I prefer no quotes.

![image](https://user-images.githubusercontent.com/31987104/144995726-82594bab-93ae-4e70-b014-b6fc32d46734.png)

Reply:
I'd like to introduce `yamllint`.
https://github.com/adrienverge/yamllint
https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.quoted_strings

Reply:
0aedfe0d88221bea0e9d036b9320a7553e191c3c